.github/workflows/third_party_notice_check.yml

```diff
@@ -0,0 +1,30 @@
+name: Check Third Party Notices File
+
+on: pull_request
+
+jobs:
+  license-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install generate-license-file
+        run: npm install -g generate-license-file
+      - uses: actions/checkout@v2
+      - name: Npm install
```

Comment:
we probably don't need a name for the steps where the name is just the literal command being run